src/controllers/midi/midicontroller.cpp

```diff
@@ -312,7 +360,7 @@ void MidiController::processInputMapping(const MidiInputMapping& mapping,
         bool found = false;
         for (auto it = m_fourteen_bit_queued_mappings.begin();
              it != m_fourteen_bit_queued_mappings.end(); ++it) {
-            if (it->first.control == mapping.control) {
+            if (std::get<ConfigKey>(it->first.control) == configKey) {
```

Comment:
I think for exception safety, it would be better to check in every loop iteration.

```suggestion
            const ConfigKey* fourteen_bit_control = std::get_if<ConfigKey>(it->first.control);
            VERIFY_OR_DEBUG_ASSERT(fourteen_bit_control == nullptr) {
                continue;
            }
            if (*fourteen_bit_control == configKey) {
```